src/main/resources/site/i18n/phrases_en.properties

```diff
@@ -10,6 +10,7 @@ modified = Modified
 notAvailableHeader = Denne siden er ikke tilgjengelig på norsk
 notAvailableBody = Finn annet innhold på norsk<br/>via vår forside
 goHome = Gå til forsiden
+close = Lukk
```

Comment:
"Close" here. This is the english phrases file. 👾 (Very easy to mix up considering the strings above)